site/docs/src/json/tenants/request.json

```diff
@@ -232,6 +249,7 @@
       "requireNumber": true,
       "validateOnLogin": false
     },
+    "state": "Active",
```

Comment:
This is not mutable using this API, it should only be documented on the response.